python/ray/serve/tests/test_pipeline_deployment_config.py

```diff
@@ -50,7 +50,7 @@ def combine(m1_output, m2_output, kwargs_output=0):
     return m1_output + m2_output + kwargs_output
 
 
-def test_deploment_options_func_class_with_class_method():
+def test_deploment_options_func_class_with_class_method(serve_instance):
```

Comment:
this shouldn't be required if we the minimal executor node uses lazy handle.

Reply:
oh there's one committed that i removed lazy handle but later realized we still need it to run dag without serve instance. let me change this back.